### ISSUE TITLE
Does not work with workspaces

### ISSUE THREAD

**Issue by pravic** [kbknapp/cargo-outdated#132]
Project structure:
```
serde/
serde/Cargo.toml
src/
Cargo.toml
```

```
[workspace]
members = [".", "serde"] 
```

> **$ cargo outdated**
error: failed to read `%TEMP%\cargo-outdated.OEWVKGM26aU4\serde\Cargo.toml`
Caused by:
  The system can not find the path specified. (os error 3)

If I remove `workspace`, it works well. But it would be better to work.

> cargo-outdated v0.7.0
> cargo 0.26.0
> rustc 1.25.0 x86_64-pc-windows-msvc

**Comment by joelgallant** [comment]
You can also, as a workaround, `cd serde` and run it from there. It would be great for this package to support `-p` to specify packages in a workspace.

**Comment by pravic** [comment]
It does not work with the current (`"."`) project - that's the point.

**Comment by joelgallant** [comment]
Right, makes sense (I didn't think about a workspace with `.` as a member like your example)
